python/doc/theory/meta_modeling/enumeration_strategy.rst

```diff
@@ -3,42 +3,61 @@
 Chaos basis enumeration strategies
 ----------------------------------
 
-| The polynomial chaos (PC) expansion allows one to obtain an explicit
-  representation of the random response :math:`\underline{Y}` of the
-  model under consideration. More precisely, the response is cast
-  as a converging series featuring orthonormal polynomials. For
-  computational purpose, it is necessary though to retain a finite
-  number of terms by truncating the expansion. First of all, a specific
-  strategy for enumerating the infinite PC series has to be defined.
-  This is the scope of the current section.
+The *functional chaos* expansion allows one to obtain an explicit
+representation of the random response :math:`\underline{Y}` of the
+model under consideration. More precisely, the response is cast
+as a converging series featuring an orthonormal basis. For
+computational purpose, it is necessary though to retain a finite
+number of terms by truncating the expansion. First of all, a specific
+strategy for enumerating the infinite PC series has to be defined.
+This is the scope of the current section.
 
 Given an input random vector :math:`\vect{X}` with prescribed
 probability density function (PDF) :math:`f_{\vect{X}}(\vect{x})`, it is
-possible to build up a *polynomial chaos* (PC) basis
-:math:`\{\psi_{\idx},\idx \in \NM\}` . Of interest is the definition of
-enumeration strategies for exploring this basis, i.e. of suitable
-*enumeration functions* :math:`\tau` from :math:`\Nset` to :math:`\NM`,
-which creates a one-to-one mapping between an integer :math:`j` and a
-multi-index :math:`\idx`.
+possible to build up a *functional chaos* multivariate basis
```

Comment:
```suggestion
possible to build up a *functional chaos* multivariate polynomial basis
```

Reply:
@jschueller Why is this conversation resolved? Do you disagree with the change @mbaudin47 proposed?

Reply:
This holds for several of @mbaudin47's comments actually.

Reply:
we should not mention polynomial here